### PR TITLE
Bugfix  FXIOS-7259 [v120] Added and implemented 'isToolbarCFREnabled' as a Nimbus flag

### PR DESCRIPTION
**Pull request by jnrahme**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7259)

## :bulb: Description
Implemented 'isToolbarCFREnabled' as a Nimbus flag to ensure the correct displaying of the toolbar CFR.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by isabelrios** [comment]
The merge will be unblocked once approved. The test failed once but then it passed in a second re-try, even though here is red bitrise should be green. Unfortunately the check here is not updated with successful retries. Only if a UI test fails 3 out of 3 will block a PR.

**Comment by isabelrios** [comment]
Another thing, if you re-run a test workflow will fail. It has to be re-run from the pipelines view -> rebuild unsuccessful workflows. But as I said before if one UI test fails just once, the PR is good to merge as Bitrise will be green, no need to re-try

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -633,7 +633,7 @@ class BrowserViewController: UIViewController,
 
     private func prepareURLOnboardingContextualHint() {
         guard contextHintVC.shouldPresentHint(),
-              featureFlags.isFeatureEnabled(.contextualHintForToolbar, checking: .buildOnly)
+              featureFlags.isFeatureEnabled(.isToolbarCFREnabled, checking: .buildOnly)
```
Comment (nbhasin2) [inline Client/Frontend/Browser/BrowserViewController.swift:636]: Slight concern here, as this is for contextual hint for toolbar but then we have an additional flag, wondering if we need to check both here `checkNimbusForContextualHintsFeature` ?

**Review by nbhasin2 — APPROVED** [review]

**Comment by nbhasin2** [comment]
@adudenamedruby For this why are there two contextual hints features configured?
There is `is-toolbar-cfr-on` and then there is toolbar-hint as part of contextual-hint-feature
This could get confusing unless you know why we have two features that kinda do the same thing? 🤔

**Comment by jnrahme** [comment]
Closing this PR ignore comments

**Pull request closed without merging**